helm/opsy/values.yaml

```diff
@@ -0,0 +1,93 @@
+# Default values for opsy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: objectrocket/opsy
+  tag: latest
```

Comment:
using latest can cause some problems, for example if you have a cluster, and Node1 is running your pod, and the app version (latest) is say 1.0 and you change later to 1.1, and then tag 1.1 as latest, if it hits Node1 again, it will not pull the image since it already believes it has the image matching tag 'latest' 

Since this helm chart is tied to a version of the application, I suggest using: .Chart.AppVersion  in the deployment/job templates, and remove this variable completely.

Also, since this chart is directly tied to the opsy release, having repository as a variable in this chart is not really needed.

Reply:
Changed to allow image tag to be set, but by default use .Chart.AppVersion. Leaving the repo variable since it's not really hurting anything and might come in handy if someone's wanting to test with a custom image.